boards/arm/pinetime_devkit0/doc/index.rst

```diff
@@ -23,102 +23,131 @@ app developers/communities.
    :align: center
    :alt: Pine64 PineTime
 
-   Board Name (Credit: <owner>)
+   PineTime Dev Kit (Credit: Pine64)
 
 Hardware
 ********
 
-The Pinetime is based on a Nordic NRF52832 chip and features:
+The PineTime is based on a Nordic NRF52832 chip and features:
 
 - 64 MHz Cortex-M4 with FPU
 - 64KB SRAM
 - 512KB on board Flash
 - 1.3 inches (33mm), 240x240 pixels display with ST7789 driver
 - 170-180mAh LiPo battery
-- XT25F32B 32Mb(4MB) SPI NOR Flash
+- XT25F32B 32Mb (4MB) SPI NOR Flash
 - CST816S Capacitive Touch
-- BMA421 Triaxial VAcceleration Sensor
+- BMA421 Triaxial Acceleration Sensor
 - HRS3300 PPG Heart Rate Sensor
 
 PineTime Port Assignment
 ========================
 
-See `Pinetime schematics`_
+See `PineTime schematics`_
 +----------------------+---------------------------------+-----------+
 | NRF52 pins           | Function                        | Direction |
 +======================+=================================+===========+
-| P0.00/XL1            | 32.768KHz –XL1                  |           |
-| P0.01/XL2            | 32.768KHz –XL1                  |           |
+| P0.00/XL1            | 32.768 kHz –XL1                 |           |
++----------------------+---------------------------------+-----------+
+| P0.01/XL2            | 32.768 kHz –XL2                 |           |
++----------------------+---------------------------------+-----------+
 | P0.02/AIN0           | SPI-SCK, LCD_SCK                | OUT       |
++----------------------+---------------------------------+-----------+
 | P0.03/AIN1           | SPI-MOSI, LCD_SDI               | OUT       |
++----------------------+---------------------------------+-----------+
 | P0.04/AIN2           | SPI-MISO                        | IN        |
++----------------------+---------------------------------+-----------+
 | P0.05/AIN3           | SPI-CE# (SPI-NOR)               | OUT       |
++----------------------+---------------------------------+-----------+
 | P0.06                | BMA421-SDA, HRS3300-SDA, TP-SDA | I/O       |
++----------------------+---------------------------------+-----------+
 | P0.07                | BMA421-SCL, HRS3300-SCL, TP-SCL | OUT       |
-| P0.08                | BMA421-INT                      | IN        |
++----------------------+---------------------------------+-----------+
+| P0.08                | BMA421-INT                      | IN        |
++----------------------+---------------------------------+-----------+
 | P0.09/NFC1           | LCD_DET                         | OUT       |
++----------------------+---------------------------------+-----------+
 | P0.10/NFC2           | TP_RESET                        | OUT       |
```

Comment:
Since these two pins are not to be used for NFC antenna but as regular GPIOs, the board would need to use `CONFIG_NFCT_PINS_AS_GPIOS=y` in its `*_defconfig` file. I haven't spotted this while reviewing the previous PR. Maybe it would be good to correct it in this PR?

Reply:
Done.